### DISCUSSION TITLE
pycapnp v2.0.0b2 released

### DISCUSSION THREAD

**Discussion by haata** [capnproto/capnproto#1854] [General]
https://pypi.org/project/pycapnp/2.0.0b2/
https://github.com/capnproto/pycapnp/releases/tag/v2.0.0b2

## v2.0.0b2 (2023-11-25)
- Fix broken test in test_load (#329)
- Update README example to async code (#331)
- Fix 'AttributeError: '_UnixSelectorEventLoop' object has no attribute 'call_soon'
- Delete and update some Python 3.7-specific todo notes
- Make a server fail early when the KJ loop is not running
- Update documentation to async code (#331) (#332)
- Fix retransmit bug for large messages causing message corruption
- Unlock the GIL for all capnp functions that do IO
- Handle exceptions from server callbacks
- Disable the use of ninja for windows builds
- DynamicCapabilityClient fix
- Make `reraise_kj_exception` available to downstream
- Support `_DynamicListReader` in `_setDynamicField`
- Fix re-raising of KjException
- Allow cancellation of all capability contexts
- Corner case for cancelled server methods that raise exceptions
- Some fixes to the magic import system

The fixes have started to slow down a bit. If there aren't any major issues I'll push out v2.0.0 in a week or two.

-HaaTa / Jacob Alexander

**Comment by christian-heusel** [comment]
Hey just a friendly poke on the release :blush:
We are currently waiting on the release of pycapnp so we can update capnp in the archlinux repositories :)

https://archlinux.org/todo/capnproto-1011-rebuild/
